lib/segment/src/data_types/named_vectors.rs

```diff
@@ -155,6 +175,7 @@ impl<'a> NamedVectors<'a> {
             match vector {
                 Vector::Dense(v) => CowVector::Dense(Cow::Owned(v)),
                 Vector::Sparse(v) => CowVector::Sparse(Cow::Owned(v)),
+                Vector::MultiDense(_) => unimplemented!("MultiDenseVector is not supported"),
```

Comment:
Is it something missing here to implement?

Reply:
Good catch, I will add the impl :+1:

Reply:
@agourlay  can you take a look on my question? I'm trying to work with spare vector and wonder how to regenerate vector without insert the same paayload again 
https://github.com/orgs/qdrant/discussions/3797

Reply:
Hey @hiepxanh, this place is not related to your question. Please keep the discussion within the expected topic.

Reply:
Yes sir, I'm sorry